devices/Vl53L0X/samples/version.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
you can get rid of this on (no need to have the version.json in the samples folder)